shell/app/modules/project/common/components/section-info-edit.tsx

```diff
@@ -54,6 +55,8 @@ class SectionInfoEdit extends React.Component<IProps, IState> {
 
   toggleModal = () => {
     this.setState({ modalVisible: !this.state.modalVisible });
+    this.props.setCanGetClusterListAndResources &&
+      this.props.setCanGetClusterListAndResources(!this.state.modalVisible);
```

Comment:
use optional call

Reply:
I mean `this.props.setCanGetClusterListAndResources?.(!this.state.modalVisible);`